### PR TITLE
Add extra documentation to sub-packages.

### PR DESCRIPTION
**Pull request by grittygrease**

### CODE REVIEW

**Comment by lziest** [comment]
LGTM

**Comment by kisom** [comment]
Style nit: the package docs should be complete sentences with punctuation. Other than that, LGTM.

**Comment by lziest** [comment]
golint is more strict. It requires the doc starts with " Package X" rather than "Package api/X"

File: api/initca/initca.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Package api/initca implements the HTTP handler for the CA initialization command
```
Comment (lziest) [inline api/initca/initca.go]: "api/initca" -> "initca"

**Pull request merged**